scripts/nolints.txt

```diff
@@ -29,7 +29,20 @@ apply_nolint CommRing.colimits.desc_morphism doc_blame
 apply_nolint CommRing.colimits.prequotient doc_blame has_inhabited_instance
 apply_nolint CommRing.colimits.relation doc_blame
 
--- algebra/category/Group.lean
+-- algebra/category/Group/colimits.lean
```

Comment:
@robertylewis What's the procedure here?
I'm inclined to say: if these should permanently not be linted, then they should get `@[nolint doc_blame]` wherever they are declared, if they should at some point get a docstring, I would rather get that docstring now.

Reply:
Ok, I now realise that these are coming from the "quasi-robot" pre-automated file. I would still suggest that either your pre-automation adds docstrings in a robot voice (so they might be pretty boring and uninformative), or otherwise add a pre-automated `@nolint` before the definitions.

Reply:
I haven't read the rest of this PR, so possibly out of context, but:

You're "not allowed" to touch this file. If your declarations are exempt from the linters (these are not AFAICT), you can add `@[nolint]` when they're defined. Otherwise you shouldn't introduce linter errors.

The exceptions when you're allowed to touch this file are if you're doing a big refactor (i.e. these already existed under different names, in which case the old names should be deleted from `nolints.txt` too) or if you're adding a new linter.

Reply:
I've added lots of docstrings, and no longer touch this file!